### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Zaid-maker**

### CODE REVIEW

**Comment by Microwave-6443** [comment]
ah yes, another unnecessary change/addition

**Comment by Jonfirexbox** [comment]
The repo doesn’t need a ReadMe we know what it is and what it does already

**Review by itzJOHv — CHANGES_REQUESTED** [review]
fix gramm**e**r

**Comment by Microwave-6443** [comment]
grammar*

**Comment by itzJOHv** [comment]
> grammar*

ik

**Comment by itzJOHv** [comment]
> grammar*

ik

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# DanBotHostingStats
+All in one monitoring software to monitor multiple systems from a easy to use web panel
```
Comment (Microwave-6443) [inline README.md:2]: 1. this is more of a discord bot
2. barely monitors anything
3. web panel developed by someone else
4. jesus christ fix your grammar

**Pull request closed without merging**